image_classification/main.js

```diff
@@ -112,6 +116,7 @@ $('#backendBtns .btn').on('change', async (e) => {
   } else {
     ui.handleBtnUI('#float16Label', true);
     ui.handleBtnUI('#float32Label', false);
+    ui.handleBtnUI('#uint8Label', false);
```

Comment:
Looks like this model is only for CPU device type, you should disable the button when it is GPU or NPU via

`ui.handleBtnUI('#uint8Label', true);`